requirements.txt

```diff
@@ -8,3 +8,5 @@ python-dateutil==2.7.3
 requests==2.20.0
 validators==0.12.6
 termcolor==1.1.0
+tqdm==4.49.0
```

Comment:
Why are we adding `tqdm`, if we are not using it?

Also, I would prefer to use `tqdm` over `progressbar2` as it is more maintained and more used. Any particular reason to use `progressbar2`?

Reply:
I saw `progressbar2` gives a lot of customization which we can use later on so I thought of using it. Not sure about `tqdm`

Reply:
I think we should use `tqdm` as it very mature and maintained. `progressbar2` suffers from issues like these mentioned here -- https://github.com/WoLpH/python-progressbar#known-issues which might create problems to users later.